src/components/panel/panel.js

```diff
@@ -649,6 +652,8 @@ function MdPanelService($rootElement, $rootScope, $injector) {
 MdPanelService.prototype.create = function(opt_config) {
   var configSettings = opt_config || {};
 
+  angular.copy(this._defaultConfigOptions, this._config);
```

Comment:
Do you need a full deep-copy? The performance of `angular.copy` is not the best.